test/remote/gateways/remote_global_collect_test.rb

```diff
@@ -12,6 +12,27 @@ def setup
     @cabal_card = credit_card('6271701225979642', brand: 'cabal')
     @declined_card = credit_card('5424180279791732')
     @preprod_card = credit_card('4111111111111111')
+    @apple_pay = network_tokenization_credit_card('4567350000427977',
+      payment_cryptogram: 'EHuWW9PiBkWvqE5juRwDzAUFBAk=',
+      month: 10,
+      year: 24,
```

Comment:
Unless there's something different about ApplePay test tokens, might be better to also make this dynamic

Reply:
Yes, you are right.